openedx/core/djangoapps/django_comment_common/comment_client/subscriptions.py

```diff
@@ -0,0 +1,50 @@
+"""
+Subscription model is used to get users who are subscribed to the main thread/post i.e.
+"""
+import logging
+
+from . import models, settings, utils
+
+log = logging.getLogger(__name__)
+
+
+class Subscription(models.Model):
+    """
+    Subscription model is used to get users who are subscribed to the main thread/post i.e.
+    """
+    # accessible_fields can be set and retrieved on the model
+    accessible_fields = [
+        '_id', 'subscriber_id', "source_id", "source_type"
+    ]
+
+    type = 'subscriber'
+    base_url = f"{settings.PREFIX}/subscriptions"
+
+    @classmethod
+    def fetch(cls, thread_id, query_params):
```

Comment:
can rename this to `fetch_threads` as we have comments that have subscriptions as well!